src/components/TestDetailsDialog/TestDetailsModal.tsx

```diff
@@ -481,6 +487,22 @@ const TestDetailsModal: React.FunctionComponent<{
         onOriginalSizeClick={setOriginalSize}
         onFitIntoScreenClick={fitStageToScreen}
       />
+      <BaseModal
+        open={applyIgnoreDialogOpen}
+        title={''}
+        submitButtonText={'Yes'}
+        onCancel={toggleApplyIgnoreDialogOpen}
+        content={
+          <Typography>
+            {`Are you sure you want to apply ignore area to all items in this build?`}
```

Comment:
drop a note that all images gonna be re-compared with new ignore area taken into account